src/transformers/models/ernie_m/configuration_ernie_m.py

```diff
@@ -0,0 +1,125 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" ErnieM model configuration"""
+# Copied from original paddlenlp repository.(https://github.com/PaddlePaddle/PaddleNLP/blob/develop/paddlenlp/transformers/ernie_m/configuration.py)
+
+from __future__ import annotations
+
+from typing import Dict
+
+from ...configuration_utils import PretrainedConfig
+
+
+ERNIE_M_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "susnato/ernie-m-base_pytorch": "https://huggingface.co/susnato/ernie-m-base_pytorch/blob/main/config.json",
+    "susnato/ernie-m-large_pytorch": "https://huggingface.co/susnato/ernie-m-large_pytorch/blob/main/config.json",
+}
+
+
+class ErnieMConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [*ErnieMModel*]. It is used to instantiate a ERNIEM
+    model according to the specified arguments, defining the model architecture. Instantiating a configuration with the
+    defaults will yield a similar configuration to that of the ERNIE-M architecture. Configuration objects inherit from
+    [*PretrainedConfig*] and can be used to control the model outputs. Read the documentation from [*PretrainedConfig*]
+    for more information. [susnato/ernie-m-base_pytorch](https://huggingface.co/susnato/ernie-m-base_pytorch)
+
+    Args:
+            vocab_size (int, *optional*):
+                Vocabulary size of *inputs_ids* in *ErnieMModel*. Also is the vocab size of token embedding matrix.
+                Defines the number of different tokens that can be represented by the *inputs_ids* passed when calling
+                *ErnieMModel*. Defaults to `250002`
+            hidden_size (int, *optional*):
+                Dimensionality of the embedding layer, encoder layers and pooler layer. Defaults to *768*.
+            num_hidden_layers (int, *optional*):
+                Number of hidden layers in the Transformer encoder. Defaults to *12*.
+            num_attention_heads (int, *optional*):
+                Number of attention heads for each attention layer in the Transformer encoder. Defaults to *12*.
+            intermediate_size (int, *optional*):
+                Dimensionality of the feed-forward (ff) layer in the encoder. Input tensors to feed-forward layers are
+                firstly projected from *hidden_size* to *intermediate_size*, and then projected back to *hidden_size*.
```

Comment:
Use backticks for code-style here and everywhere else in this docstring:
```suggestion
                firstly projected from `hidden_size` to `intermediate_size`, and then projected back to `hidden_size`.
```